jboss-kie-wildfly-common/tests/bats/jboss-kie-wildfly-common.bats

```diff
@@ -12,6 +12,23 @@ export CONFIG_FILE=$JBOSS_HOME/standalone/configuration/standalone-openshift-log
 
 setup() {
   cp $BATS_TEST_DIRNAME/../../../jboss-eap-config-openshift/EAP7.4.0/added/standalone-openshift.xml $CONFIG_FILE
+
+  mkdir -p  ${JBOSS_HOME}/domain
```

Comment:
don't add this to the setup, otherwise it will be created every test.

Reply:
I moved in a dedicated function used in the 2 test cases